### ISSUE TITLE
poetry-exec-plugin isn't compatible with poetry 1.5.0

### ISSUE THREAD

**Issue by jnussbaum** [keattang/poetry-exec-plugin#14]
The newest release of poetry (1.5.0) makes it impossible to execute a command with poetry-exec-plugin:

```
poetry exec -vvv some-command
(...long stack trace...)
  AttributeError

  'PyProjectTOML' object has no attribute '_file'

  at ~/Library/Application Support/pypoetry/venv/lib/python3.9/site-packages/poetry_exec_plugin/plugin.py:39 in handle
       35│         ),
       36│     ]
       37│ 
       38│     def handle(self) -> Any:
    →  39│         pyproject_folder_path = self.poetry.pyproject._file.path.parent
       40│         pyproject_data = self.poetry.pyproject.data
       41│ 
       42│         cmd_name = self.argument("cmd")
       43│         cmd = (
```

I tested it out locally, and the fix seems to be very easy: Just change in line 39: `self.poetry.pyproject.file.path.parent`

**Comment by jnussbaum** [comment]
@keattang Is there any hope that this will be fixed soon? Looking at how old the issues and PRs in this repo are, I'm a bit pessimistic about the future of poetry-exec-plugin... 
I very much like poetry-exec-plugin - it would be sad if it slowly dies...

**Comment by keattang** [comment]
Hi all, a pull request is always appreciated. I also intend to take a look this weekend.

**Comment by jnussbaum** [comment]
I created a branch locally, and the tests pass 😄 . But unfortunately I don't have permissions to push.
However, I spotted that in the GitHub CI, you test your plugin against a rather old version of poetry: https://github.com/keattang/poetry-exec-plugin/blob/main/.github/workflows/test.yaml#L27

**Comment by fhcj2** [comment]
This fix would be very much appreciated indeed! That's the only thing stopping my team from upgrading poetry to the latest version (1.5), and there're new features in it we're looking forward to. We could workaround it but it'd be nice if it came in as an official fix!

**Comment by keattang** [comment]
Thanks all for identifying the issue and @masuwo3 for his fix. This should now be fixed if you upgrade to 0.3.6
